spec/behaviour_raker_rspec.rb

```diff
@@ -0,0 +1,140 @@
+require 'spec_helper'
```

Comment:
You HAD a problem, not "was having" XD
I really don't know, how to resolve your problem that i required to use sudo for running your tests, and can't find the answer just looking on this screenshot.
Try to google it more